app/lib/common/widgets/chat/convo_card.dart

```diff
@@ -58,15 +63,76 @@ class _ConvoCardState extends ConsumerState<ConvoCard> {
                 latestMessage: latestMsg,
               )
             : const SizedBox.shrink(),
-        trailing: latestMsg != null
-            ? _TrailingWidget(
-                // controller: receiptController,
-                room: widget.room,
-                latestMessage: latestMsg,
-                activeMembers: activeMembers,
-                userId: client!.userId().toString(),
-              )
-            : const SizedBox.shrink(),
+        trailing: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: [
+            latestMsg != null
+                ? _TrailingWidget(
+                    // controller: receiptController,
+                    room: widget.room,
+                    latestMessage: latestMsg,
+                    activeMembers: activeMembers,
+                    userId: client!.userId().toString(),
+                  )
+                : const SizedBox.shrink(),
+            mutedStatus.valueOrNull == true
+                ? Expanded(
+                    child: MenuAnchor(
+                      builder: (
+                        BuildContext context,
+                        MenuController controller,
+                        Widget? child,
+                      ) {
+                        return IconButton(
+                          padding: EdgeInsets.zero,
+                          iconSize: 14,
+                          onPressed: () {
+                            if (controller.isOpen) {
+                              controller.close();
+                            } else {
+                              controller.open();
+                            }
+                          },
+                          icon: const Icon(Atlas.bell_dash_bold),
+                        );
+                      },
+                      menuChildren: [
+                        MenuItemButton(
+                          child: const Text('Unmute'),
+                          onPressed: () async {
+                            final room = await ref.read(
+                              maybeRoomProvider(widget.room.getRoomIdStr())
+                                  .future,
+                            );
+                            if (room == null) {
+                              // ignore: use_build_context_synchronously
+                              customMsgSnackbar(
```

Comment:
Can we agree on using ```EasyLoading``` dialogs which would allow us to not require context and potentially adding these ignore lines?